chia/full_node/coin_store.py

```diff
@@ -176,10 +174,6 @@ async def get_coin_records(self, names: Collection[bytes32]) -> List[CoinRecord]
         return coins
 
     async def get_coins_added_at_height(self, height: uint32) -> List[CoinRecord]:
-        coins_added: Optional[List[CoinRecord]] = self.coins_added_at_height_cache.get(height)
-        if coins_added is not None:
-            return coins_added
-
```

Comment:
I don't think we use this function when validating blocks or transactions, so the cache probably wouldn't affect validation.

Reply:
Yes seems like this code is mainly used by the bluebox code unfortunately. I am not totally sure since it is difficult to trace all paths.